ukbot/filters.py

```diff
@@ -803,16 +866,28 @@ def fetch(self):
         logger.info('SparqlFilter: Initialized with %d articles', len(self.page_keys))
 
     def add_linked_articles(self, site, item_var):
+        wikidata_items = self.get_competition_wikidata_ids(self.tpl, site)
+
+        # If possible wikidata_items is none then there is no need for SPARQL query
+        if not wikidata_items:
```

Comment:
But what if it returned no Wikidata items because of an error, or because fiwiki-tools is down for some unrelated reason? In that case, I think it should fall back to the old VALUES-less SPARQL query, instead of not running the query at all, like now.

Essentially: I'd like the script to distinguish between "get_competition_wikidata_ids() worked as it should and returned no items" and "get_competition_wikidata_ids() had an exception and therefore returned no items". Does that make sense?